ui/src/azure-account.api.d.ts

```diff
@@ -4,9 +4,9 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { Event } from 'vscode';
-import { ServiceClientCredentials } from 'ms-rest';
-import { AzureEnvironment } from 'ms-rest-azure';
-import { SubscriptionModels } from 'azure-arm-resource';
+import { ServiceClientCredentials } from '@azure/ms-rest-js';
+import { Environment } from '@azure/ms-rest-azure-env/lib/azureEnvironment'
```

Comment:
So this file is copied from here: https://github.com/microsoft/vscode-azure-account/blob/master/src/azure-account.api.d.ts

Are the old types (`AzureEnvironment` and `ServiceClientCredentials`) compatible with the new types (`Environment ` and `ServiceClientCredentials`)? Have we talked to Christof about this at all?

Reply:
I haven't talked to him. Will talk to him. 

Brings me to the question: Why was this file copied? This adds problems with versioning and documentation (there wasn't any warning in the file for instance)

Reply:
I agree versioning/docs would be ideal, but it's just not a pressing issue for the following reasons:
1. It's a `.d.ts` file, so it's only there for the types. It never gets compiled or run from this repo
1. It hasn't changed since 2017 when it was originally added to this repo
1. It's supposed to be backwards compatible

Also, we intentionally put this file in the shared ui package so that each individual extension would not have to deal with it.

Reply:
I had a look at ui/package.json, and there's no reference to the azure-account library. This makes sense to me now. 
As for backwards compatibility it looks like we might have to break it for this change.